### ISSUE TITLE
_test.go files are now shown in coverage list

### ISSUE THREAD

**Issue by Cosby86** [uartois/sonar-golang#62]
Hello all
 i just updated my sonar plugin to the latest RC (rc12). Despite I added those 2 lines into config file

sonar.test.inclusions=**/**_test.go
sonar.sources.inclusions=**/**.go

In my coverage file list now are shown _test,go files too

**Comment by thibaultfalque** [comment]
hi @Cosby86 
Could you test with this configuration ? 
```   
sonar.sources=./
sonar.tests=./
sonar.test.inclusions=**/**_test.go
sonar.sources.inclusions=**/**.go
```

**Comment by Cosby86** [comment]
Thank you for the answer. Not _test files are not considered but I have in the list packages where _test.go file are not presents. In the previous version of the plugin only the coverage of those packages where _test.go file is present were listed.

**Comment by thibaultfalque** [comment]
Yes we have changed the coverage computation. See #56, #60 and #61. If there is not have _test.go in package the sources files of package are covered with 0%.

**Comment by Cosby86** [comment]
It sound good. Is there any way to exclude a package from coverage mesurement?

**Comment by thibaultfalque** [comment]
I will ask sonarqube to know how use the sonar.coverage.exclusions property (https://docs.sonarqube.org/display/SONAR/Analysis+Parameters)

**Comment by thibaultfalque** [comment]
@danielleberre we can use the method _doesNotMatchPathPatterns_ of _FilePredicates_ for exclude the patterns of property sonar.coverage.exclusions: http://javadocs.sonarsource.org/6.2.1/apidocs/org/sonar/api/batch/fs/FilePredicates.html#doesNotMatchPathPatterns-java.lang.String:A-

**Comment by thibaultfalque** [comment]
Hello @Cosby86 

Do you have the problem with the latest version ? If not, could you close this issue ?
